LICENSE

```diff
@@ -1,6 +1,19 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Justin Ellis
+*** Copyright Notice ***
+PTMCMCSampler Copyright (c) 2015 to 2021, Justin Ellis
+All rights reserved.
+
+PTMCMCSampler Copyright (c) 2022, Mark Forrer on behalf of
```

Comment:
I'm fine with adding this although really we should probably add others as well since I haven't really done any development on this code for several years. @AaronDJohnson, @paulthebaker what do you think?